code-gen-projects/schema/struct_with_fields.isl

```diff
@@ -3,7 +3,8 @@ type::{
  fields: {
     A: string,
     B: int,
-    C: { element: string }
+    C: { element: string },
+    D: float,
```

Comment:
(🗺️ PR tour) Added this schema field to test for double value in Java.

Reply:
Do you need to update the rust code gen tests to account for this new field?

Reply:
No, it already includes roundtrip tests so it considers this new field in that.

Reply:
Right... the rust tests are generated. It doesn't need to be this PR, but we should add some non-generated tests. The generated tests are useful, but they won't be able to catch some classes of problems (e.g. when an anonymous type is renamed unexpectedly) because the tests are created from the same code-generation logic as the generated code.